frozen-abi/src/stable_abi.rs

```diff
@@ -5,6 +5,6 @@ pub trait StableAbi: Sized {
     where
         Standard: rand::distributions::Distribution<Self>,
     {
-        rng.gen::<Self>()
+        rng.r#gen::<Self>()
```

Comment:
Would be nice to add lints for 2024 compatibility to lib.rs to make sure this is sufficient.